dali/operators/decoder/audio/audio_decoder.h

```diff
@@ -47,10 +47,34 @@ class AudioDecoderBase {
   }
 
   /**
-   * @brief Decode audio data and store it in the supplied buffer
-   * @return Number of (multi-channel) samples actually read
+   * @brief Determines whether the particular decoder can seek frames
    */
-  virtual ptrdiff_t Decode(span<char> raw_output) = 0;
+  virtual bool CanSeekFrames() const = 0;
```

Comment:
Is there a decoder, that wouldn't be able to seek frames? If not, maybe we could remove this function?
If we would really want to optionally add seeking, IMHO it should be covered by a decorator. But I'd prefer assuming, that we can always seek frames